### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by dennis-every**
Completed the following changes:
- Add contact form HTML section
- Add CSS styles to contact-form
- Wrap contact-form elements in ul/li
- Update form action with form-spree-url
- Update contact-form background image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -324,5 +324,67 @@ <h2 id="connect">Let's Connect</h2>
...
+                  ></textarea>
+                </label>
+              </li>
+              <li>
+                <button id="btn-contact-form" type="submit">
+                  Get in touch
+                </button>
+              </li>
+            </ul>
+          </form>
```
Comment (abdulhamiid) [inline index.html:385]: | Your design  | Figma design |
| ------------- | ------------- |
| image  | image 

As it was stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) we must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Please kindly place all the background images in their positions as they appear in the Figma design. as that will show our strength 💪 in UI design by following the given design template closely.

Please consider fixing the left and right margins of the section as shown above by using equal left and right spacing to match the Figma design.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @dennis-every 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

### Please let us update the README FILE 👏

As a way to document our work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). we should include all main headers from the [README template](https://github.com/microverseinc/readme-template) and customize them accordingly. please 🙏 kindly consider adding more description to your README file using [MICROVERSE README template](https://github.com/microverseinc/readme-template) as a way to make the README file more descriptive and professional 💪

[📖 About the Project](https://github.com/microverseinc/readme-template#about-project)
[🛠 Built With](https://github.com/microverseinc/readme-template#built-with)

- [Tech Stack](https://github.com/microverseinc/readme-template#tech-stack)
- [Key Features](https://github.com/microverseinc/readme-template#key-features)
- [🚀 Live Demo](https://github.com/microverseinc/readme-template#live-demo)

[💻 Getting Started](https://github.com/microverseinc/readme-template#getting-started)

- [Setup](https://github.com/microverseinc/readme-template#setup)
- [Prerequisites](https://github.com/microverseinc/readme-template#prerequisites)
- [Install](https://github.com/microverseinc/readme-template#install)
- [Usage](https://github.com/microverseinc/readme-template#usage)
- [Run tests](https://github.com/microverseinc/readme-template#run-tests)
- [Deployment](https://github.com/microverseinc/readme-template#triangular_flag_on_post-deployment)

[👥 Authors](https://github.com/microverseinc/readme-template#authors)
[🔭 Future Features](https://github.com/microverseinc/readme-template#future-features)
[🤝 Contributing](https://github.com/microverseinc/readme-template#contributing)
[⭐️ Show your support](https://github.com/microverseinc/readme-template#support)
[🙏 Acknowledgements](https://github.com/microverseinc/readme-template#acknowledgements)
[❓ FAQ](https://github.com/microverseinc/readme-template#faq)
[📝 License](https://github.com/microverseinc/readme-template#license)

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dennis-every** [comment]
Made the following changes per code review:
- Updated readme
- Centered contact form elements and contained background image to align with Figma design
- Added a black bar at the bottom of the page

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```
Comment (abdulhamiid) [inline README.md]: Good job implementing the requested changes 👍

As we are required to customize the README template according to our project, please let's remove the Microverse template logo 🙏. Optionally, you can replace it with your own logo if they have one 🤗.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @dennis-every 👋 , 

Great work on making the changes requested in the previous review 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (dennis-every) [reply README.md]: Hi @abdulhamiid, thanks for your comments, however as this is a Microverse project I would really like to keep that Microverse logo there as it nicely shows the organization I'm doing this project for. Please let me know if you have any other comments.

**Comment by abdulhamiid** [comment]
Hi @dennis-every, you have mentioned a good point, however it is part of Microverse **terms and conditions** that every student using her README template should remove the Microverse template logo. and they can replace it with their own logo if they have one.

Thanks 🤗.

File: README.md (hunk 2 above)
Comment (abdulhamiid) [reply README.md]: Hi @dennis-every, you have mentioned a good point, however it is part of Microverse **rules** that every student using her README template should remove the Microverse template logo. and they can replace it with their own logo if they have one.

Thanks 🤗.

Comment (dennis-every) [reply README.md]: Hi @abdulhamiid, thanks for your reply and explanation - makes sense. I have removed the Microverse logo from the readme. Thanks 🤗

Comment (abdulhamiid) [reply README.md]: Cheers and Happy coding! 👏👏👏

**Review by Debas-31 — APPROVED** [review]
> ## Hi @dennis-every and @kensteph   :wave:, 

# Status: Approved :heavy_check_mark:  :heavy_check_mark:  :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Project validation functionality :heavy_check_mark: 
- Figma template usage :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**